File: VB/ExtendedWinAdapter/ExtendedWinAdapter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.ExpressApp.EasyTest.WinAdapter
Imports DevExpress.EasyTest.Framework
Imports AdditionalCommands

Namespace ExtendedAdapters
	Public Class ExtendedWinAdapter
		Inherits WinAdapter
		Public Overrides Sub RegisterCommands(ByVal registrator As DevExpress.EasyTest.Framework.IRegisterCommand)
			MyBase.RegisterCommands(registrator)
			registrator.RegisterCommand("FillDateTimeValue", GetType(FillDateTimeValueCommand))
		End Sub
	End Class
End Namespace

File: CS/ExtendedWinAdapter/ExtendedWinAdapter.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.ExpressApp.EasyTest.WinAdapter;
using DevExpress.EasyTest.Framework;
using AdditionalCommands;

namespace ExtendedAdapters
{
    public class ExtendedWinAdapter : WinAdapter
    {
        public override void RegisterCommands(DevExpress.EasyTest.Framework.IRegisterCommand registrator)
        {
            base.RegisterCommands(registrator);
            registrator.RegisterCommand("FillDateTimeValue", typeof(FillDateTimeValueCommand));
        }
    }
}
